test_orgs.txt

```diff
@@ -0,0 +1,25 @@
+Organization.create(name: 'Allegheny Center Alliance Church', active: true)
```

Comment:
This is a deprecated change. This file should not exist any more, and since they're moved into seeds.rb

Reply:
Bump-- let's delete this file.